### ISSUE TITLE
Changing root to /usr/local breaks boxen entirely

### ISSUE THREAD

**Issue by indirect** [boxen/puppet-brewcask#49]
It seems that puppet will not allow the latest releases of `homebrew` and `brewcask` to both be used at the same time:

```
$ bin/librarian-puppet show
brewcask (0.0.8)
homebrew (2.1.0)

$ script/boxen
[...]
Error: Duplicate declaration: File[/usr/local] is already declared in file 
/opt/boxen/repo/shared/homebrew/manifests/init.pp:53; cannot redeclare at 
/opt/boxen/repo/shared/brewcask/manifests/init.pp:17 on node finn.local
```

😢

**Comment by jacobbednarz** [comment]
I did see another issue about the duplicate path resources being declared. Let me see if I can find it and get it sorted.

**Comment by indirect** [comment]
This fixed it for me: https://github.com/indirect/puppet-brewcask/commit/0d26da49d67eab89ba1749bfa63ab487127014fb

**Comment by jacobbednarz** [comment]
I _think_ this is 👌  because homebrew now manages and installs cask too.

**Comment by indirect** [comment]
That is my understanding as well. Caskroom is tapped by default. 👍

**Comment by jacobbednarz** [comment]
@mikemcquaid any objections to merging the homebrew and brewcask puppet modules?

**Comment by MikeMcQuaid** [comment]
Go for it, makes sense now.

**Comment by jacobbednarz** [comment]
I've opened boxen/puppet-homebrew#102 to get this fixed. Thanks for the push @indirect!

**Comment by jacobbednarz** [comment]
@indirect I've push up a more permanent fix to this issue if you wanna check it out - https://github.com/boxen/puppet-brewcask/pull/50. I'm running it internally at the moment ensuring we iron out any bugs and I'd ❤️  your feedback.

**Comment by indirect** [comment]
@jacobbednarz looks like a good solution to me 👍

**Comment by jacobbednarz** [comment]
(hopefully) fixed via #50
